src/ints/mouse.cpp

```diff
@@ -701,12 +697,16 @@ static void Mouse_Reset()
 	Mouse_AfterNewVideoMode(false);
 	Mouse_SetMickeyPixelRate(8,16);
 
-	mouse.mickey_x = 0;
-	mouse.mickey_y = 0;
+	mouse.mickey_x   = 0;
+	mouse.mickey_y   = 0;
+	mouse.buttons    = 0;
+    mouse.wheel      = 0;
+    mouse.cute_mouse = false;
 
-	mouse.buttons = 0;
+    mouse.last_wheel_moved_x = 0;
+    mouse.last_wheel_moved_y = 0;
 
```

Comment:
wonky indentation

Reply:
Used the script to correct all the formatting.